node_modules/dotenv/config.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
```

Comment:
did you make this file explicitly? or was this autogenerated?
if explicitly, can you explain to me why this is necessary?